Formula/youplot.rb

```diff
@@ -0,0 +1,49 @@
+class Youplot < Formula
+  desc "Command-line tool that draw plots on the terminal"
+  homepage "https://github.com/red-data-tools/YouPlot/"
+  url "https://github.com/red-data-tools/YouPlot/archive/v0.4.5.tar.gz"
+  sha256 "aa7339139bc4ea9aa0b2279e4e8052fde673a60ad47e87d50fde06626dc2b3c3"
+  license "MIT"
+
+  uses_from_macos "ruby"
+
+  resource "unicode_plot" do
+    url "https://rubygems.org/downloads/unicode_plot-0.0.5.gem"
+    sha256 "91ce6237bca67a3b969655accef91024c78ec6aad470fcddeb29b81f7f78f73b"
+  end
+
+  resource "enumerable-statistics" do
+    url "https://rubygems.org/downloads/enumerable-statistics-2.0.7.gem"
+    sha256 "eeb84581376305327b31465e7b088146ea7909d19eb637d5677e51f099759636"
+  end
```

Comment:
looks like these resources did not get built individually?

Reply:
You are correct. I will fix it in the next commit.

Reply:
looks good!